xrpl4j-model/src/test/java/org/xrpl/xrpl4j/model/transactions/XrpCurrencyAmountTest.java

```diff
@@ -64,4 +67,38 @@ public void toXrp() {
     assertThat(XrpCurrencyAmount.ofDrops(MAX_XRP_IN_DROPS - 1).toXrp()).isEqualTo(new BigDecimal("99999999999.999999"));
   }
 
+  @Test
+  public void plusXrp() {
+    assertThat(
+      XrpCurrencyAmount.ofDrops(HALF_XRP_IN_DROPS)
+        .plus(XrpCurrencyAmount.ofDrops(HALF_XRP_IN_DROPS))
+    ).isEqualTo(XrpCurrencyAmount.ofDrops(ONE_XRP_IN_DROPS));
+
+    assertThat(
+      XrpCurrencyAmount.ofXrp(XrpCurrencyAmount.ofDrops(1L).toXrp())
+        .plus(XrpCurrencyAmount.ofXrp(XrpCurrencyAmount.ofDrops(0L).toXrp()))
+    ).isEqualTo(XrpCurrencyAmount.ofDrops(1L));
+  }
+
+  @Test
+  public void minusXrp() {
+    assertThat(
```

Comment:
Can we add a unit test to cover the case where subtraction yields a negative number?